build/buildsage.sh

```diff
@@ -15,40 +15,10 @@
 # limitations under the License.
 #
 echo "Bulding Sage.jar..."
-rm -rf TempClassFiles
-mkdir TempClassFiles
-rm -rf TempBuildImages
-mkdir TempBuildImages
-cd TempBuildImages
-rm -rf images
-mkdir images
-cd images
-cp ../../../images/SageTV/images/*.gif .
-cp ../../../images/SageTV/images/*.png .
-mkdir studio
-cd studio
-cp ../../../../images/SageTV/images/studio/*.gif .
+
+# now uses gradle because there a dynamic dependencies for JOGL
```

Comment:
That shouldn't be true...there should be no JOGL dependencies for the Sage.jar file.